metadata-load/src/test/java/uk/ac/ebi/ampt2d/metadata/importer/api/SraObjectsImporterThroughApiTest.java

```diff
@@ -122,8 +120,8 @@ public void importStudy() throws Exception {
         assertEquals(21, referenceSequenceRepository.count());
         assertEquals(25, sampleRepository.count());
 
-        expectedException.expect(InvalidReferenceSequenceException.class);
-        sraObjectImporter.importStudy("ERP111092");
+        expectedException.expect(AnalysisWithoutReferenceSequenceException.class);
+        sraObjectImporter.importStudy("ERP001296");
```

Comment:
Yes, because we're checking for an [expected exception](https://www.mkyong.com/unittest/junit-4-tutorial-2-expected-exception-test/):

```
expectedException.expect(AnalysisWithoutReferenceSequenceException.class);
sraObjectImporter.importStudy("ERP001296");
```

The “assertion” here (`expectedException.expect`) is actually before the tested statement (`sraObjectImporter.importStudy`)